quic-client/src/nonblocking/quic_client.rs

```diff
@@ -540,11 +510,7 @@ impl QuicClientConnection {
         addr: SocketAddr,
         connection_stats: Arc<ConnectionCacheStats>,
     ) -> Self {
-        let client = Arc::new(QuicClient::new(
-            endpoint,
-            addr,
-            QUIC_MAX_UNSTAKED_CONCURRENT_STREAMS,
```

Comment:
Why is max_parallel_streams removed?

Reply:
@lijunwangs The logic that read this variable was removed entirely so the setting no longer does anything. 
The QUIC client now sends each stream sequentially. This is faster and simpler than the fragmentation that the parallel setting created. The trick to make this work is that the client no longer waits for acknowledgements.